pkgs/servers/clickhouse/default.nix

```diff
@@ -46,7 +55,6 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DENABLE_TESTS=OFF"
-    "-DUSE_INTERNAL_LLVM_LIBRARY=OFF"
```

Comment:
Why is this removed? We do not want to vendor LLVM.

Reply:
Was removed in the previous PR, I've added it back and updated the flags to keep up with upstream's requirements

Reply:
It does take less time to compile when you're using the non-vendored LLVM though :^)

Reply:
True, but I reckon most people compiling it from source won't care much about a one-time 10ish minute saving (at least on my hardware)

The majority of users will be grabbing it from cache.nixos.org

I'm open to making this configurable with a default to `OFF`